src/pages/Product/Top/Top.scss

```
@@ -0,0 +1,181 @@
+@import '../../../styles/common.scss';
+
+.top-top {
+  // height: 629px;
+  border-bottom: 1px solid lightgrey;
+  color: $color-black;
+  .top-backimg {
+    position: relative;
+    // height: 394px;
+    .backimg-container {
+      display: flex;
+      flex-direction: row;
+      .backimg-left {
+        flex: 1 1 0%;
+        background-color: rgb(24, 23, 20);
+        // width: 440.5px;
+      }
```

```suggestion
.top-top {
  // height: 629px;
  border-bottom: 1px solid lightgrey;
  color: $color-black;
  
  .top-backimg {
    position: relative;
    // height: 394px;
    
    .backimg-container {
      display: flex;
      flex-direction: row;
      
      .backimg-left {
        flex: 1 1 0%;
        background-color: rgb(24, 23, 20);
        // width: 440.5px;
      }
```
위와 같이 새로운 selector가 시작될 때는 줄바꿈을 통해 가독성을 높여주세요 :)

---

해당 부분 뿐만 아니라 전체적으로 수정해주세요!

---

확인했습니다. 감사합니다!